SmartEVSE-3/src/main.cpp

```diff
@@ -4195,6 +4207,10 @@ bool handle_URI(struct mg_connection *c, struct mg_http_message *hm,  webServerR
 
         doc["mains_meter"]["import_active_energy"] = round((float)MainsMeter.Import_active_energy / 100)/10; //in kWh, precision 1 decimal
         doc["mains_meter"]["export_active_energy"] = round((float)MainsMeter.Export_active_energy / 100)/10; //in kWh, precision 1 decimal
+        if (MainsMeter.Type == EM_HOMEWIZARD_P1) {
+            auto host = getCachedHomeWizardHost();
```

Comment:
We try to avoid the:

int getVariable()
return X;
void setVariable(int set)
X=set;
return;
constructions; because of the time loops we are forced to have a lot of global/static variables anyways, and it only adds extra lines of code if you add two global function to address a variable that then you can make local.

Please address cachedHomeWizardHost directly; perhaps better rename it to HomeWizardHost since any variable can be seen as "cached" ?